### ISSUE TITLE
official ums docker-image for arm64

### ISSUE THREAD

**Issue by seiha011** [UniversalMediaServer/UniversalMediaServer#4770]
Hi Guys,
I found your docker image here.  https://hub.docker.com/r/universalmediaserver/ums/tags
Unfortunately only for amd64. Are there plans to provide an official docker image for arm64 as well?

**Comment by tfonias74** [comment]
I have the same question but I see no responses

**Comment by SubJunk** [comment]
I think this is a good idea, I'm just not sure how to do it

**Comment by skiesewetter-intershop** [comment]
That can not be a big problem as there is already an linux/arm64 binary release.

**Comment by SubJunk** [comment]
Yeah I agree, I don't think it will be hard, it just needs someone to do it. Maybe you can help?
